src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -112,6 +111,11 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
+        # Delete log files to prevent interference from the current run on the next execution.
+        try:
+            shutil.rmtree(ProcessPoolConstants.PROCESS_LOG_PATH)
+        except Exception as e:
+            bulk_logger.warning(f"Failed to delete the folder, exception: {e}")
 
     @property
     def is_timeout(self):
```

Comment:
Seems you will delete the dir eventually, then why not use shutil.rmtree to remove the folder directly

Reply:
fixed